compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaConstantSymbol.java

```diff
@@ -162,6 +162,11 @@ private String toStringVal(Object obj, BType valType) {
         return String.valueOf(obj);
     }
 
+    @Override
+    public TypeSymbol originalType() {
```

Comment:
We don't have to add `originalType()` API in constant symbols right?

Reply:
@dulajdilshan, `BallerinaConstantSymbol` class implements the `ConstantSymbol` interface which extends the `SingletonTypeSymbol` interface. Therefore, when a new API method is exposed via the `SingletonTypeSymbol` interface, its implementation should also be made available in the `BallerinaConstantSymbol` class. 

As per a discussion held with @mohanvive, it was decided to proceed with providing the `broaderTypeDescriptor` for constants which aligns with singleton's original type.

Reply:
Ah okay. I noticed `ConstantSymbol` extends `SingletonType`.